awxkit/awxkit/cli/client.py

```diff
@@ -170,9 +170,9 @@ def parse_resource(self, skip_deprecated=False):
             if formatted:
                 print(utils.to_str(formatted), file=self.stdout)
         else:
-            self.parser.print_help()
-
-            if six.PY2 and not self.help:
+            if six.PY3:
+                self.parser.print_help()
+            elif six.PY2 and not self.help:
```

Comment:
This was a bug that made the help text print twice in PY2 if you did `awx -h` (whoops)